src/api/hub.ts

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+export default class HubGroupAPI {
+  private readonly defaultBaseUrl = 'https://63d6be0e-bd49-4c76-8fe5-195bb7cf88a5.mock.pstmn.io';
```

Comment:
This is not useful here, please remove it

Reply:
fyi, as per our discussion last week, default base url will be retained. So I'll resolve this conversation